src/phpcs/Production/Sniffs/Abstract/MethodSniff.php

```diff
@@ -51,10 +51,10 @@ protected function hasMethodDocBlock(PHP_CodeSniffer_File $sniffedFile, $indexOf
     }
 
     /**
-     * Prüft, ob das Token an $index ein Whitespace ist.
+     * Checks if the token is a whitespace.
      *
-     * @param PHP_CodeSniffer_File $sniffedFile durchsuchte Datei
-     * @param int                  $index Position des aktuellen Tokens in der Tokens-Liste
+     * @param PHP_CodeSniffer_File $sniffedFile
+     * @param int                  $index
```

Comment:
... position of current token in token list.

Please repeat for all other occurrences.